lms/static/scripts/tsconfig.json

```diff
@@ -12,6 +12,15 @@
     "target": "ES2020",
     "useUnknownInCatchVariables": false
   },
-  "include": ["frontend_apps/**/*.js", "frontend_apps/**/*d.ts"],
-  "exclude": ["frontend_apps/**/test/*.js"]
+  "include": ["**/*.js", "**/*.ts", "**/*.tsx"],
+  "exclude": [
+    "new-application-instance.js",
```

Comment:
`new-application-instance.js` is going away very soon. See https://github.com/hypothesis/lms/pull/4975.